src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -364,7 +373,7 @@ public sealed partial class InputsDialogValidationContext
     {
         public required System.Threading.CancellationToken CancellationToken { get { throw null; } init { } }
 
-        public required System.Collections.Generic.IReadOnlyList<InteractionInput> Inputs { get { throw null; } init { } }
+        public required InteractionInputCollection Inputs { get { throw null; } init { } }
```

Comment:
Does the `InputsDialogValidationContext` need an Experimental attribute on it? I think we missed it last time.

cc @JamesNK

Reply:
#11520